docs/build-windows.md

```diff
@@ -2,16 +2,25 @@
 
 Windows builds can be created using:
 
-- MSVC compiler, Visual Studio 2022 IDE suite, and vcpkg to provide dependencies
-  *(recommended)*.
+- MSClang compiler, Visual Studio 2022 IDE suite, and vcpkg to provide
+dependencies. This is the fully-supported toolchain used to
+create release builds.
 - The Clang or GCC compilers using the Meson buildsystem running within the
   MSYS2 environment to provide dependencies.
 
-
 ## Build using Visual Studio
 
 1. Install Visual Studio Community 2022: <https://visualstudio.microsoft.com/>.
```

Comment:
While you're at it, here's a better link:

https://visualstudio.microsoft.com/vs/community/